mllib/src/test/scala/org/apache/spark/mllib/evaluation/RegressionMetricsSuite.scala

```diff
@@ -22,91 +22,115 @@ import org.apache.spark.mllib.util.MLlibTestSparkContext
 import org.apache.spark.mllib.util.TestingUtils._
 
 class RegressionMetricsSuite extends SparkFunSuite with MLlibTestSparkContext {
+  val obs = List[Double](77, 85, 62, 55, 63, 88, 57, 81, 51)
+  val eps = 1E-5
 
   test("regression metrics for unbiased (includes intercept term) predictor") {
```

Comment:
I thought when the intercept is included, the stats should be correct. Why do you change this test case?